### DISCUSSION TITLE
New dark mode feature

### DISCUSSION THREAD

**Discussion by ISODev23** [Sketchware-Pro/Sketchware-Pro#1080] [Q&A]
I would like to suggest the possibility of incorporating a dark mode in the Sketchware application. I believe adding this functionality would be an excellent option to enhance the visual comfort of users, especially during prolonged usage.

Dark mode not only provides a more comfortable visual experience in low-light environments but can also reduce eye fatigue. Many users would appreciate the flexibility to choose between light and dark modes according to their personal preferences.

I greatly appreciate the work you do in the development of Sketchware, and I believe that the inclusion of a dark mode would be a valuable enhancement. Could you please consider this suggestion in future updates?

**Comment by j-px** [comment]
You don't have to give reasons why a dark mode would be nice to have, lol. Anyway, that's part of making the app Material You-themed, so it might be available for all users "soon".
